manifests/a/astral-sh/uv/0.4.20/astral-sh.uv.locale.en-US.yaml

```diff
@@ -0,0 +1,62 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: astral-sh.uv
+PackageVersion: 0.4.20
+PackageLocale: en-US
+Publisher: Astral
+PublisherUrl: https://github.com/astral-sh/uv
+PublisherSupportUrl: https://github.com/astral-sh/uv/issues
+Author: uv contributors
+PackageName: uv
+PackageUrl: https://github.com/astral-sh/uv
+License: Apache-2.0

```

Comment:
This result was automatically generated, and the authorized name does not match the link provided.